scrapinghub/hubstorage/collectionsrt.py

```diff
@@ -32,7 +32,7 @@ def _allows_mpack(self, path=None):
         :param path: None, tuple or string
 
         """
-        if not MSGPACK_AVAILABLE:
+        if not MSGPACK_AVAILABLE or not self.client.use_msgpack:
```

Comment:
quick question, why not set `self.use_msgpack = MSGPACK_AVAILABLE and use_msgpack` in init and check `if not self.client.use_msgpack` only?